server/services/omuCaselist.test.ts

```diff
@@ -0,0 +1,175 @@
+import { add, sub } from 'date-fns'
+import OmuCaselist from './omuCaselist'
+import LicenceStatus from '../enumeration/licenceStatus'
+import LicenceType from '../enumeration/licenceType'
+import { ManagedCase } from '../@types/managedCase'
+import Container from './container'
+
+const cases = [
+  {
+    deliusRecord: { offenderId: 1 },
+    licences: [{ type: LicenceType.AP, status: LicenceStatus.SUBMITTED }],
+    nomisRecord: {
+      status: 'ACTIVE IN',
+      confirmedReleaseDate: '2023-04-25',
+      conditionalReleaseDate: '2023-02-19',
+      restrictedPatient: false,
+    },
+  },
+  {
+    deliusRecord: { offenderId: 2 },
+    licences: [{ type: LicenceType.AP, status: LicenceStatus.IN_PROGRESS }],
+    nomisRecord: {
+      status: 'ACTIVE IN',
+      confirmedReleaseDate: '2023-04-25',
+      conditionalReleaseDate: '2023-02-19',
+      restrictedPatient: false,
+    },
+  },
+  {
+    deliusRecord: { offenderId: 3 },
+    licences: [{ type: LicenceType.AP, status: LicenceStatus.ACTIVE }],
+    nomisRecord: {
+      status: 'OUT',
+      confirmedReleaseDate: '2023-04-25',
+      conditionalReleaseDate: '2023-02-19',
+      restrictedPatient: false,
+    },
+  },
+  {
+    deliusRecord: { offenderId: 4 },
+    licences: [{ type: LicenceType.AP, status: LicenceStatus.VARIATION_IN_PROGRESS }],
+    nomisRecord: {
+      status: 'ACTIVE OUT',
+      confirmedReleaseDate: '2023-04-25',
+      conditionalReleaseDate: '2023-02-19',
+      restrictedPatient: false,
+    },
+  },
+  {
+    deliusRecord: { offenderId: 5 },
+    licences: [{ type: LicenceType.AP, status: LicenceStatus.VARIATION_SUBMITTED }],
+    nomisRecord: {
+      status: 'ACTIVE OUT',
+      confirmedReleaseDate: '2023-04-25',
+      conditionalReleaseDate: '2023-02-19',
+      restrictedPatient: false,
+    },
+  },
+] as unknown as Container<ManagedCase>
```

Comment:
this shouldn't be a cast, should wrap in a Container(). A Container isn't an array

Reply:
done